packages/core/package.nls.json

```
@@ -143,7 +143,7 @@
     "AWS.command.refreshCdkExplorer": "Refresh CDK Explorer",
     "AWS.command.cdk.help": "View CDK Documentation",
     "AWS.command.ec2.openTerminal": "Open terminal to EC2 instance...",
```

maybe:

```suggestion
    "AWS.command.ec2.openTerminal": "Connect Terminal to EC2 instance...",
```

p.s. vscode UX guidelines ask for `Title Case` in command/action descriptions